src/bin/ion/commands/beta/generate/templates/java/class.templ

```diff
@@ -1,19 +1,19 @@
-{% if import %}
-import ion_data_model.{{ import.name | upper_camel }};
-{% endif %}
+package {{ namespace }};
```

Comment:
(🗺️ PR tour) These changes were found during roundtrip testing locally. 
- Adds namespace declaration statement.
- Adds `ArrayList` import.
- Add changes to use `camel` casing for field names in Java.